### PR TITLE
Feat/stock view

### PR DESCRIPTION
**Pull request by JaroJenaro**
Erste lauffähige Teil Code

### CODE REVIEW

File: backend/src/main/java/de/iav/backend/controller/StockController.java [hunk 1]
```diff
@@ -1,4 +1,49 @@
...
+    public List<User> getPetsBySpecificSearch(@RequestParam(required = false) String firstName, @RequestParam(required = false) String lastName ){
+
+        if(firstName != null)
+        {
+            return stockService.findAllByFirstNameEqualsIgnoreCase(firstName);
+        }
+        else{
+            return stockService.findAllByLastNameEqualsIgnoreCase(lastName);
+        }
+    }*/
```
Comment (HoumanMohammadi) [inline backend/src/main/java/de/iav/backend/controller/StockController.java:48]: not comments in code jaro. wann willst du das lernen!

File: backend/src/main/java/de/iav/backend/controller/TransactionController.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    public List<User> getPetsBySpecificSearch(@RequestParam(required = false) String firstName, @RequestParam(required = false) String lastName ){
+
+        if(firstName != null)
+        {
+            return stockService.findAllByFirstNameEqualsIgnoreCase(firstName);
+        }
+        else{
+            return stockService.findAllByLastNameEqualsIgnoreCase(lastName);
+        }
+    }*/
```
Comment (HoumanMohammadi) [inline backend/src/main/java/de/iav/backend/controller/TransactionController.java:55]: please delete

File: backend/src/main/java/de/iav/backend/controller/TransactionController.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+
+/*
+    @GetMapping("/set")
+    public List<Stock> getPetById(){
+        return transactionService.setStockByRepository();
+    }
+
+
```
Comment (HoumanMohammadi) [inline backend/src/main/java/de/iav/backend/controller/TransactionController.java:44]: delete!

File: backend/src/main/java/de/iav/backend/service/StockService.java [hunk 4]
```diff
@@ -29,6 +40,13 @@ public List<Stock> getStockByISIN(String isin){
...
+    public List<Stock> setStockByRepository(){
+        if (stockRepository.findAll().size() == 0)
+            return fillDataWithStocks();
+        else
+            return stockRepository.findAll();
+
+    }
     /*
     public Stock createOwner(Stock student){
         return stockRepository.save(student);
```
Comment (HoumanMohammadi) [inline backend/src/main/java/de/iav/backend/service/StockService.java:52]: delete

**Review by HoumanMohammadi — APPROVED** [review]
reviewd

**Review by HoumanMohammadi — APPROVED** [review]
lgtm

**Review by ErumSh — APPROVED** [review]
lgtm

**Pull request merged**